### PR TITLE
Add DB examples to Readme

### PR DESCRIPTION
**Pull request by felixfbecker**
Adds some async operations to the examples for #581.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -59,7 +59,9 @@ app.use((ctx, next) => {
 // response
 
 app.use(ctx => {
-  ctx.body = 'Hello World';
+  return someDbOperation().then(result => {
```
Comment (stojanovic) [inline Readme.md]: You don't need `{}` and `return`.

File: Readme.md [hunk 2]
```diff
@@ -59,7 +59,9 @@ app.use((ctx, next) => {
...
+    ctx.body = 'Hello World, here is your result: ' + result;
```
Comment (stojanovic) [inline Readme.md]: Use template strings here:
```js
ctx.body = `Hello World, here is your result: ${result}`;
```

**Comment by tejasmanohar** [comment]
Personally, -1 / :-1:. would leave this out of the README since it's getting application-specific. That said, if we do include it, I recommend we change `someDbOperation` to `someAsyncOperation` (or `someAsyncOp`) to make it as generic as possible and avoid questions about what DB operation? :P

**Comment by stojanovic** [comment]
:+1:

File: Readme.md (hunk 2 above)
Comment (tejasmanohar) [inline Readme.md]:
```javascript
res => ctx.body = `Hello World, here is your result: ${result}`
```

Comment (stojanovic) [inline Readme.md]: Yes, I really like arrow functions because of that :)

**Comment by juliangruber** [comment]
-1 from me as well, it's out of scope for koa

**Comment by yanickrochon** [comment]
Why not putting all these examples and extra documentation in the wiki? Keep the README light and simple.

**Comment by MarkHerhold** [comment]
:-1: for the README.
:+1: for putting it in the Wiki under a common use-cases section.

Maybe include a link to the wiki page in the README?

**Comment by felixfbecker** [comment]
I agree with the comments here.

**Comment by ericmdantas** [comment]
Pretty much what @MarkHerhold said

**Comment by dead-horse** [comment]
please put this example to https://github.com/koajs/examples :)

**Pull request closed without merging**